templates/components/common/social-links.html

```diff
@@ -1,11 +1,25 @@
 {{#if social_media}}
     <ul class="socialLinks socialLinks--alt">
         {{#each social_media}}
-            <li class="socialLinks-item">
-                <a class="icon icon--{{display_name}}" href="{{url}}" target="_blank" rel="noopener">
-                    <svg><use xlink:href="#icon-{{display_name}}" /></svg>
-                </a>
-            </li>
+            {{#if display_name '===' 'facebook'}}
```

Comment:
do we know how many `display_name` options are available at all?

Reply:
@bc-alexsaiannyi I took all possible options (at the moment) from the Control Panel settings.

Reply:
I don't see that specific `display_name` has any influence on showing `components/common/social-links-item` component or not. It seems as the reason to show social-links-item component depends on `social_media` condition on higher level. So if we remove checking  `display_name`  will this affect us at all?

Reply:
@bc-alexsaiannyi thanks, I changed my previous approach to solve this problem.